recipes/sella/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "sella" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Sella-{{ version }}.tar.gz
+  sha256: dfe262042c87d90fbb69252e9559f53faa52b0c2f41404cf991910d1429c784d
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - numpy
```

Comment:
Is numpy actually needed in host?

If it is needed you probably need to modify run https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy

Reply:
Numpy is needed in post, I pinned it correctly in run now.